incubator/virtualcluster/pkg/vn-agent/server/server_test.go

```diff
@@ -413,9 +420,13 @@ func getPodName(name, namespace string) string {
 	return name + "_" + namespace
 }
 
+func getTenantNamespace(tenantName, namespace string) string {
```

Comment:
How about getTenantNamespace -> getEffectiveNamespace, also add a comment to this method.

Reply:
updated